packages/angular_devkit/build_angular/src/builders/browser-esbuild/index.ts

```diff
@@ -258,6 +268,7 @@ function createCodeBundleOptions(
     mainFields: ['es2020', 'browser', 'module', 'main'],
     conditions: ['es2020', 'es2015', 'module'],
     resolveExtensions: ['.ts', '.tsx', '.mjs', '.js'],
+    metafile: true,
```

Comment:
Should this be false, when stats are disabled?

Reply:
The info will be used for the console stat output as well as bundle budgets so we'll always need to generate the data internally.  It is only written to disk if the stats option is enabled though.